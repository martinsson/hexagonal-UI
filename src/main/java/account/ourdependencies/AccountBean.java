package account.ourdependencies;

public class AccountBean {

    public String getSiret() {
        // TODO Auto-generated method stub
        return null;
    }

    public Object getPassword() {
        // TODO Auto-generated method stub
        return null;
    }

    public String getCondGeneMobile() {
        // TODO Auto-generated method stub
        return null;
    }

    public String getCondGeneInternet() {
        // TODO Auto-generated method stub
        return null;
    }

    public String getEmail() {
        // TODO Auto-generated method stub
        return null;
    }

}
