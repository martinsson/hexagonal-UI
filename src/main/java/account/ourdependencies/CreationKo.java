package account.ourdependencies;

public class CreationKo extends TemplateMail {

    public CreationKo(AccountBean accountBean,
            String contactEmailBO) {
        // TODO Auto-generated constructor stub
    }

}
