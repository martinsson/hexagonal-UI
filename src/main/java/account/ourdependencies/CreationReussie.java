package account.ourdependencies;

public class CreationReussie extends Email {

    public CreationReussie(ElementsInfoForMailCreation elementsInfoForMail,
            String contactEmailBO) {
        // TODO Auto-generated constructor stub
    }

}
