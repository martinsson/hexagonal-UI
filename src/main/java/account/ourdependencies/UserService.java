package account.ourdependencies;


public class UserService {

    public void createAccount(UserInfo siuUserInfo, Object password, String condGeneInternet,
            String condGeneMobile) {
    }

    public boolean isEmailAlreadyUsed(String email)  {
        return false;
    }

}
