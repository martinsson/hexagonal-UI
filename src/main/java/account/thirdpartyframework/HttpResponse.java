package account.thirdpartyframework;

import java.util.HashMap;
import java.util.Map;


public class HttpResponse {

    Map<String, String> attributes = new HashMap<String, String>();
    private Map<String, String> renderParameters = new HashMap<String, String>();
    private String redirect;
    public void setAttribute(String attribute, String value) {
        attributes.put(attribute, value);
    }

    public void setRenderParameter(String param, String value) {
        renderParameters .put(param, value);
    }

    public void sendRedirect(String url) {
        redirect = url;
    }

    public String getAttribute(String string) {
        return attributes.get(string);
    }

    @Override
    public String toString() {
        return "HttpResponse [attributes=" + attributes + ", renderParameters=" + renderParameters + ", redirect=" + redirect + "]";
    }
    
    

}
