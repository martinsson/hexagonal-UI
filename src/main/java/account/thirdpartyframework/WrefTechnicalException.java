package account.thirdpartyframework;

public class WrefTechnicalException extends Exception {

}
