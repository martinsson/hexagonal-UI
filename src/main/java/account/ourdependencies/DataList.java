package account.ourdependencies;

import account.thirdpartyframework.PortalException;
import account.thirdpartyframework.SystemException;

public class DataList {

    public boolean findAndCheckSiret(String siret)  throws SystemException, PortalException {
        // TODO Auto-generated method stub
        return false;
    }

}
