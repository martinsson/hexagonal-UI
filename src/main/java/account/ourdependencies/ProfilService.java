package account.ourdependencies;

import account.thirdpartyframework.WrefTechnicalException;

public class ProfilService {

    public ModelProfil findProfilWithSiret(String siret) throws WrefTechnicalException{
        // TODO Auto-generated method stub
        ModelProfil modelProfil = new ModelProfil(siret);
        return modelProfil;
    }

}
