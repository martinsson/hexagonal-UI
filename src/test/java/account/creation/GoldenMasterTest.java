package account.creation;

import static org.mockito.Mockito.doReturn;
import static org.mockito.Mockito.doThrow;
import static org.mockito.Mockito.spy;

import java.io.IOException;

import org.approvaltests.legacycode.LegacyApprovals;
import org.junit.Test;

import account.ourdependencies.AccountBean;
import account.ourdependencies.DataList;
import account.ourdependencies.ProfilService;
import account.ourdependencies.UserService;
import account.thirdpartyframework.HttpResponse;
import account.thirdpartyframework.WrefTechnicalException;

public class GoldenMasterTest {

    private static final Object[] BOTH_BOOLEAN_VALS = {true, false};

    @Test
    public void non_regression() throws Exception {
        String[] email = {"me@home.fr", null, "", "without_AT", "no_dots"};
        String[] siret = {"52448536400011", "", "123", "52448536400011524485364000115244853640001152448536400011"};
        Object[] siretException = BOTH_BOOLEAN_VALS;
        Object[] restricted = BOTH_BOOLEAN_VALS;
        Object[] emailUsed = BOTH_BOOLEAN_VALS;
        LegacyApprovals.LockDown(this, "callACC", email, siret, siretException, restricted, emailUsed);
        
    }
    
    public String callACC(String email, String siret, Boolean siretException, Boolean restricted, Boolean emailUsed) throws IOException {
        ProfilService profilService = spy(new  ProfilService());
        UserService userService = spy(new UserService());
        DataList datalist = spy(new DataList());
        
        doReturn(emailUsed).when(userService).isEmailAlreadyUsed(email);
//        if (siretException)
//            System.out.println("");
////            doThrow(new WrefTechnicalException()).when(datalist).findAndCheckSiret(siret);
//        else
            doReturn(restricted).when(datalist).findAndCheckSiret(siret);
        
        AccountCreationController controller = new AccountCreationController(new AccountService(userService, profilService, datalist));
        HttpResponse response = new HttpResponse();
        controller.doAction(new AccountBean(null, email, siret), null, response);
        
        return response.toString();
        // TODO Auto-generated method stub

    }

}
