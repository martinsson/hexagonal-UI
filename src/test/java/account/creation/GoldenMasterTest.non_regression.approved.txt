[me@home.fr, 52448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, true, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, false, true, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, true, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, false, false, true] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, true, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, false, true, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, , true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, , true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, , true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, , true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, , true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 123, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 123, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 123, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 123, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 123, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[null, 52448536400011524485364000115244853640001152448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[, 52448536400011524485364000115244853640001152448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, true, false, false] = HttpResponse [attributes={}, renderParameters={}, redirect=someUrl] 
[me@home.fr, 52448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[null, 52448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[, 52448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[without_AT, 52448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[no_dots, 52448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[me@home.fr, , false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[null, , false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[, , false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[without_AT, , false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[no_dots, , false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[me@home.fr, 123, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[null, 123, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[, 123, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[without_AT, 123, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[no_dots, 123, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[me@home.fr, 52448536400011524485364000115244853640001152448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[null, 52448536400011524485364000115244853640001152448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[, 52448536400011524485364000115244853640001152448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[without_AT, 52448536400011524485364000115244853640001152448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
[no_dots, 52448536400011524485364000115244853640001152448536400011, false, false, false] = HttpResponse [attributes={}, renderParameters={action=redirect}, redirect=null] 
